broker/Gemfile

```diff
@@ -11,6 +11,7 @@ gem 'systemu'
 gem 'mongoid'
 gem 'bson'
 gem 'bson_ext'
+gem 'net-ssh'
```

Comment:
Thanks guys - I was starting to wonder about that after I submitted the PR. I'll get the Requires line added.